src/java/org/apache/cassandra/cql3/restrictions/StatementRestrictions.java

```diff
@@ -835,6 +835,32 @@ public boolean hasRegularColumnsRestrictions()
     {
         return hasRegularColumnsRestrictions;
     }
+
+    /**
+     * Checks if the query is a full partitions selection.
+     * @return {@code true} if the query is a full partitions selection, {@code false} otherwise.
+     */
+    private boolean queriesFullPartitions()
+    {
+        return !hasClusteringColumnsRestrictions() && !hasRegularColumnsRestrictions();
+    }
+
+    /**
+     * Determines if the query should return the static content when a partition without rows is returned (as a
+     * result set row with null for all other regular columns.)
+     *
+     * @return {@code true} if the query should return the static content when a partition without rows is returned, {@code false} otherwise.
```

Comment:
Nit: we can split this line:
```suggestion
     * @return {@code true} if the query should return the static content when a partition without rows is returned,
     * {@code false} otherwise.
```